administration/templates/administration/approve.html

```diff
@@ -10,12 +10,13 @@
 
 
 {% block content %}
-<div class="uu-inner-container">
+<div class="uu-container">
     <div class="col-12">
         <h1>
             {% trans 'administration:approve:header' %}
         </h1>
-        <form method="post">{% csrf_token %}
+        <form method="post" class="uu-form uu-form-no-gap">
+            {% csrf_token %}
             <p>{% transformat 'administration:approve:text' object %}</p>
             <div class="button" style="display: inline-block">
                 <a href="{% url 'administration:home' %}" >
```

Comment:
This should be rewritten to Bootstrap buttons, as well as the submit button below it